### DISCUSSION TITLE
Where is drivers/i2s.h

### DISCUSSION THREAD

**Discussion by nmaron** [marcel-licence/esp32_basic_synth#82] [Q&A]
I get compile errors and it appears that BCLK_PIN and related aren't defined. I thought they might be in drivers/i2s.h but that directory and files is not to be found. Can I get some direction on this. I am trying to use the ESP32 Audio Kit ES8388 board.

**Comment by marcel-licence** [comment]
Hello,

BCLK_PIN can be defined by yourself depending on your setup and the connection from ESP32 to the audio codec. If you are using for example the audio kit you can set a specific define in the config.h wich will finally include one of the pre-defined board settings:
https://github.com/marcel-licence/ML_SynthTools/tree/main/src/boards

**Reply by nmaron** [reply]
I’ll try that. Thanks.

**Reply by marcel-licence** [reply]
Okay, I am looking forward for your feedback
